x-pack/plugins/infra/public/components/asset_details/tabs/overview/tile.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiIcon,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiI18n,
+  EuiToolTip,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import type { Action } from '@kbn/ui-actions-plugin/public';
+import { useLensAttributes } from '../../../../hooks/use_lens_attributes';
+import type { HostsLensMetricChartFormulas } from '../../../../common/visualizations';
+import { LensWrapper } from '../../../../common/visualizations/lens/lens_wrapper';
+import { buildCombinedHostsFilter } from '../../../../utils/filters/build';
+import { TooltipContent } from '../../../../common/visualizations/metric_explanation/tooltip_content';
+import type { KPIGridProps } from './kpi_grid';
+
+export interface KPIChartProps {
+  title: string;
+  subtitle?: string;
+  trendLine?: boolean;
+  backgroundColor: string;
+  type: HostsLensMetricChartFormulas;
+  decimals?: number;
+  toolTip: string;
+}
+
+const MIN_HEIGHT = 150;
+
+export const Tile = ({
+  title,
+  type,
+  backgroundColor,
+  toolTip,
+  decimals = 1,
+  trendLine = false,
+  nodeName,
+  dateRange,
+  dataView,
+}: KPIChartProps & KPIGridProps) => {
```

Comment:
nit: 

```suggestion
export interface KPIChartProps, KPIGridProps {
  title: string;
  subtitle?: string;
  trendLine?: boolean;
  backgroundColor: string;
  type: HostsLensMetricChartFormulas;
  decimals?: number;
  toolTip: string;
}

const MIN_HEIGHT = 150;

export const Tile = ({
  title,
  type,
  backgroundColor,
  toolTip,
  decimals = 1,
  trendLine = false,
  nodeName,
  dateRange,
  dataView,
}: KPIChartProps) => {
```

Reply:
I moved the config and the type to a common folder so I now import both interfaces and still have `KPIChartProps & KPIGridProps` is that fine?